spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefromfields">
+          <h1>CalendarDateToISO ( _calendar_, _fields_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateToISO performs implementation-defined processing to convert _fields_, which is a Record that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record.</p>
```

Comment:
are these AOs only in 402, and not in 262 at all? Should there be a 262 "base implementation"?

Reply:
Yes, they're 402 only. The corresponding methods on the Calendar prototype have base implementation, but they don't need these AOs anyway.

Reply:
Hmm, ok - I'm mainly thinking that anything in 402 that isn't strictly coupled to Intl data would be really nice to have in 262, so there's a smaller delta between a sans-Intl and with-Intl implementation. No concrete suggestions here tho.